src/lib/playlists.ts

```diff
@@ -283,6 +285,7 @@ export async function loadPlaylist(slug: string, token?: string) {
   })
 
   const {playlist} = await graphQLClient.request(query, variables)
+  const courseMeta = await loadCourse(playlist.id)
 
-  return playlist
+  return {...playlist, ...courseMeta}
```

Comment:
idea: just merge these together, eventually the idea of a "course" is a combination of a finite set of resource types. egghead backend focuses on media primarily, other aspects are migrated to sanity